src/yieldOptions/providers/protocols/stargate.ts

```diff
@@ -0,0 +1,146 @@
+import { Address } from "viem";
+import { Yield } from "src/yieldOptions/types.js";
+import { Clients, IProtocol, getEmptyYield } from "./index.js";
+import axios from "axios";
+
+const STARGATE_LP_STAKING_ADDRESS = { 1: "0xB0D502E938ed5f4df2E681fE6E419ff29631d62b", 42161: "0xeA8DfEE1898a7e0a59f7527F076106d7e44c2176" }
+
+// @dev Make sure the keys here are correct checksum addresses
+const STG_ADDRESS = { 1: "0xAf5191B0De278C7286d6C7CC6ab6BB8A73bA2Cd6", 42161: "0x6694340fc020c5E6B96567843da2df01b2CE1eb6" }
+
+interface Pool {
+  chain: string;
+  project: string;
+  underlyingTokens: string[];
+  apy: number;
+}
+
+const networkMap = {
+  1: "Ethereum",
+  42161: "Arbitrum",
+  137: "Polygon"
+}; // TODO is there some viem native function that we can use here?
```

Comment:
I don't think so but we have the network map in one of the helper files https://github.com/Popcorn-Limited/vaultcraft-sdk/blob/main/src/lib/helpers.ts#L44